packages/webpack/src/plugin.js

```diff
@@ -590,6 +612,8 @@ module.exports = LavaMoatPlugin
  * @property {boolean} [HtmlWebpackPluginInterop] - Add a script tag to the html
  *   output for lockdown.js if HtmlWebpackPlugin is in use
  * @property {RegExp} [inlineLockdown] - Prefix the matching files with lockdown
+ * @property {RegExp} [unlockedChunksUnsafe] - Give matching chunks an unsafe
```

Comment:
re: usability. is use of `RegExp` typical for matching filepaths in webpack plugins? if so, then this is fine, but otherwise globs make the easy cases easier.

Reply:
1. It's quite common
2. AFAIR, we don't have other options that are flexible enough other than forcing the user to resolve a path to a file that's yet to be created

We could revisit this decision, I don't have the mental context in which I made this decision.

Reply:
I remember - a popular way to set up webpack is so that it adds a checksum or a cachebuster to the end of the script filename in dist. that's why we wanted regex